src/NLU.DevOps.Luis.Shared/LuisSettings.cs

```diff
@@ -36,7 +36,7 @@ public LuisSettings(LuisApp appTemplate)
         /// </summary>
         /// <param name="prebuiltEntityTypes">Prebuilt entity types.</param>
         public LuisSettings(IReadOnlyDictionary<string, string> prebuiltEntityTypes)
-            : this(null, prebuiltEntityTypes)
+            : this(null, prebuiltEntityTypes, null)
```

Comment:
>prebuiltEntityTypes [](start = 25, length = 19)

you don't want to add a check to see if this is null and create an empty dict instead? Same for appTemplate above?

Reply:
null check occurs in the constructor below.